components/GreyButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
add some quick comment to the top to describe what this is and where this is used.